cloud-controller-manager/do/loadbalancers.go

```diff
@@ -218,6 +220,10 @@ func (l *loadBalancers) EnsureLoadBalancer(ctx context.Context, clusterName stri
 	}
 
 	if lb.Status != lbStatusActive {
+		if lb.Status == lbStatusNew {
```

Comment:
I'd move the new `if` statement outside of the pre-existing `if` statement to ease readability.

Reply:
Sure, I've moved it above the existing `if`.